<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>121</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Operaciones</string>
    </property>
    <widget class="QRadioButton" name="radio_input">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_suma">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Suma</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_resta">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Resta</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_division">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Division</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_multiplicacion">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Multiplicacion</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_sen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Seno</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_ln">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Logartimo Natural</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_absoluto">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Valor Absoluto</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_potencia">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Potencia</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_cos">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Coseno</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_tan">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Tangente</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_minimo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Minimo</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_maximo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Maximo</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_output">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="push_add">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Agregar Bloque</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Input:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="conexiones">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Realizar Conexiones</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_boton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar Botones</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Todas las entradas conectadas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
